<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSETODBC</class>
 <widget class="QDialog" name="CSETODBC">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>334</width>
    <height>388</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>数据源设置</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>311</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QPushButton" name="QpbtnOK">
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="QpbtnClose">
      <property name="text">
       <string>退出</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>311</width>
     <height>301</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>MySql</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>68</y>
       <width>208</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>90</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>数据库地址：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="QlineIP">
        <property name="minimumSize">
         <size>
          <width>110</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>110</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_3">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>119</y>
       <width>208</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>数据库名称：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="QlineName"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_4">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>29</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>90</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>配置选项：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox">
        <item>
         <property name="text">
          <string>数据库驱动</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>数据源驱动</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_5">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>221</y>
       <width>208</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>数据库密码：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="QlinePass"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_6">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>170</y>
       <width>208</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>数据库用户：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="QlineUser"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>MsSql</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget_7">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>203</y>
       <width>208</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>数据库密码：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="QlinePass1"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_8">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>101</y>
       <width>208</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>数据库名称：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="QlineName1"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_9">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>152</y>
       <width>208</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>数据库用户：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="QlineUser1"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_10">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>208</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>90</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>数据库地址：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="QlineIP1">
        <property name="minimumSize">
         <size>
          <width>110</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>110</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>comboBox</tabstop>
  <tabstop>QlineIP</tabstop>
  <tabstop>QlineName</tabstop>
  <tabstop>QlineUser</tabstop>
  <tabstop>QlinePass</tabstop>
  <tabstop>QpbtnOK</tabstop>
  <tabstop>QpbtnClose</tabstop>
  <tabstop>QlineIP1</tabstop>
  <tabstop>QlineName1</tabstop>
  <tabstop>QlineUser1</tabstop>
  <tabstop>QlinePass1</tabstop>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
